packages/fxa-shared/metrics/glean/web/index.ts

```diff
@@ -69,7 +69,6 @@ export const eventsMap = {
     ageInvalid: 'reg_age_invalid',
     marketing: 'reg_marketing_engage',
     changeEmail: 'reg_change_email_link_click',
-    whyWeAsk: 'reg_why_do_we_ask_link_click',
```

Comment:
Should we keep the metric in the yaml file if it will no longer used? Not sure how that impacts previously collected data.

Reply:
Hm not sure? @chenba is there a process/preference for this?

Reply:
I believe we can remove them.